uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/leave-request-api_test.js

```diff
@@ -409,10 +398,35 @@ define([
 
         it('returns error', function () {
           promise.catch(function (result) {
-            expect(result).toEqual(errorObject);
+            expect(result).toBe(errorMessage);
           });
         });
       });
     });
+
+    describe('isManagedBy()', function () {
+
+      beforeEach(function () {
+        spyOn(LeaveRequestAPI, 'sendGET').and.callThrough();
+        promise = LeaveRequestAPI.isManagedBy(jasmine.any(String), jasmine.any(String));
+      });
+
+      afterEach(function () {
+        $httpBackend.flush();
+      });
+
+      it('calls endpoint', function () {
+        promise.then(function () {
+          expect(LeaveRequestAPI.sendGET).toHaveBeenCalledWith('LeaveRequest',
+            'isManagedBy', jasmine.any(Object));
```

Comment:
Let's reference real parameters here, you should test not only if the endpoint is called, but with what parameters. This is especially important since the `isManagedBy` accepts two separate params (contact id, request id) which are then used to build a `params` object